shared/src/main/java/gov/hhs/cdc/trustedintermediary/metadata/MetaDataEntry.java

```diff
@@ -0,0 +1,17 @@
+package gov.hhs.cdc.trustedintermediary.metadata;
+
+import java.util.Date;
+
+/** An instance of a metadata event to be used for internal troubleshooting of messages */
+public class MetaDataEntry {
```

Comment:
You can make this a `record`.  Checkout the `AuthRequest.java`.  It allows you to write a simple immutable Java class easily.

Reply:
done.